web_ui/src/shared/components/dataset-import-progress/circular-progress/circular-progress.component.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2022-2025 Intel Corporation
+// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+import { useMemo } from 'react';
+
+import { ColorValue } from '@react-types/shared/src/dna';
+
+interface CircularProgressProps {
+    percentage: number;
+    size?: number;
+    labelFontSize?: number;
+    strokeWidth?: number;
+    labelFontColor?: ColorValue;
+    backStrokeColor?: ColorValue;
+    color?: ColorValue;
+    hasError?: boolean;
+    checkMarkOnComplete?: boolean;
+    checkMarkSize?: number;
+    checkMarkColor?: ColorValue | string;
+}
+
+export const CircularProgress = ({
```

Comment:
Doesn't it need tests?

Reply:
We can add it later. I just moved files around